package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "powerbi-visuals-api",
-  "version": "4.4.0",
+  "version": "5.0.0",
```

Comment:
in comments it says version 4.5.0 but here you changed to 5.0.0 (?)
also - please change the changelog.md file to track the version changes